connexion/app.py

```diff
@@ -85,8 +86,13 @@ def common_error_handler(exception):
         """
         :type exception: Exception
         """
+        # If we have an instance of problem exception we can use the title to render a more complete title.
+        if isinstance(exception, ProblemException):
+            return problem(title=exception.title, detail=exception.description, status=exception.code)
+
         if not isinstance(exception, werkzeug.exceptions.HTTPException):
             exception = werkzeug.exceptions.InternalServerError()
+
         return problem(title=exception.name, detail=exception.description, status=exception.code)
```

Comment:
Instead of the previous `if` from line 90. Please do this here:
```python
problem_title = getattr(exception, 'title', exception.name)
return problem(title=problem_title, detail=exception.description, status=exception.code)
```